lab-tuple-set-dict/your-code/.ipynb_checkpoints/challenge-1-checkpoint.ipynb

```diff
@@ -0,0 +1,350 @@
+   "source": [
+    "#### Add `tup1` and `tup2` into `tup3` using the `+` operator.\n",
+    "\n",
+    "Then print `tup3` and check if `tup3` equals to `tup`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "('I', 'r', 'o', 'n', 'h', 'a', 'c', 'k')\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Your code here\n",
+    "tup3=tup1+tup2\n",
+    "print(tup3)\n",
+    "print(tup==tup3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Count the number of elements in `tup1` and `tup2`. Then add the two counts together and check if the sum is the same as the number of elements in `tup3`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "8\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Your code here\n",
+    "cont1=0\n",
+    "cont2=0\n",
+    "cont3=0\n",
+    "for elem in tup1:\n",
+    "    cont1+=1\n",
+    "for elem in tup2:\n",
+    "    cont2+=1\n",
+    "for elem in tup3:\n",
+    "    cont3+=1\n",
+    "cont=cont1+cont2\n",
+    "print(cont3==cont)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### What is the index number of `\"h\"` in `tup3`?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "4\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Your code here\n",
+    "cont=0\n",
+    "for ele in tup3:\n",
+    "    if ele=='h':\n",
+    "        print(cont)\n",
+    "    cont+=1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Now, use a FOR loop to check whether each letter in the following list is present in `tup3`:\n",
+    "\n",
+    "```\n",
+    "letters = [\"a\", \"b\", \"c\", \"d\", \"e\"]\n",
+    "```\n",
+    "\n",
+    "For each letter you check, print `True` if it is present in `tup3` otherwise print `False`.\n",
+    "\n",
+    "*Hint: you only need to loop `letters`. You don't need to loop `tup3` because there is a Python operator `in` you can use. See [reference](https://stackoverflow.com/questions/17920147/how-to-check-if-a-tuple-contains-an-element-in-python).*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Your code here\n",
+    "letters = [\"a\", \"b\", \"c\", \"d\", \"e\"]\n",
+    "for elem in letters:\n",
+    "    for elem2 in tup3:\n",
+    "        if elem==elem2:\n",
+    "            print(True)"
+   ]
```

Comment:
No era necesario entrar en la segunda lista: 

for elem in letters:
    if elem in tup3:
        print(True)